src/DropdownAlignment.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+/**
+ * Alignment for the dropdown component.
+ *
+ * - `END`: Aligns the dropdown menu to the end of the dropdown toggle.
```

Comment:
Let's move these descriptions to constants.

Reply:
done.